fvm/evm/stdlib/contract.go

```diff
@@ -242,12 +159,19 @@ func newInternalEVMTypeEncodeABIFunction(
 				return true
 			})
 
-			encodedValues, err := arguments.Pack(values...)
+			hexData, err := arguments.Pack(values...)
 			if err != nil {
 				panic(abiEncodingError{})
 			}
 
-			return interpreter.ByteSliceToByteArrayValue(inter, encodedValues)
+			encodedValues := interpreter.ByteSliceToByteArrayValue(inter, hexData)
+
+			invocation.Interpreter.ReportComputation(
+				environment.ComputationKindEVMEncodeABI,
+				uint(encodedValues.Count()),
```

Comment:
nice change.

Reply:
I guess this must have snuck back in by some conflict when merging in the master branch. This was just a naive implementation that I made as a first take :sweat_smile:
What we actually want in production, is the implementation that is currently on master.

Reply:
resynced master, there shouldn't be any changes to stdlib package now.